vulns/json-logic/PYSEC-2023-209.yaml

```diff
@@ -1,41 +0,0 @@
-id: PYSEC-2023-209
-modified: 2023-10-20T16:31:18.877419Z
-published: 2023-03-05T19:15:00Z
```

Comment:
Hi, thanks for this change.

Note that the the correct way to remove this advisory is to add a `withdrawn` field. See https://github.com/pypa/advisory-database/blob/ad36176d8503d666b4e83e105bc08a52dbba44ae/vulns/redis/PYSEC-2023-73.yaml#L7.